x-team/xp-ui
# Remove Storyshots

**Release Type:** *Dev improvements*

Fixes https://github.com/x-team/xp/issues/1900

## Description

Remove Storyshots.

## Checklist

> - [ ] Snapshots tests passed (`npm run jest`) -- special attention to this one (see the note below)
> - [ ] if any snapshots have been changed, verify that component still works and looks as expected and update the changed snapshot -- special attention to this one as well (see the note below)

**Note:** I had to remove the `jest` test from `npm t` because we still don't have any test. But I kept the Jest lib and all its configurations because I believe we'll want to have tests soon. When that time comes we just need to append the `npm run jest` back to the `npm test` script.

```
> xp-ui@2.0.0 jest /Users/bernardo/Sites/x-team/xp-ui
> jest --roots=src

No tests found
In /Users/bernardo/Sites/x-team/xp-ui
  184 files checked.
  testMatch: **/__tests__/**/*.js?(x),**/?(*.)+(spec|test).js?(x) - 0 matches
  testPathIgnorePatterns: /node_modules/ - 184 matches
Pattern:  - 0 matches
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! xp-ui@2.0.0 jest: `jest --roots=src`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the xp-ui@2.0.0 jest script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bernardo/.npm/_logs/2019-11-07T11_37_39_461Z-debug.log
npm ERR! Test failed.  See above for more details.
```

## Steps to Test or Reproduce

1. Run `npm t`
1. Observe that there's no Storyshot tests anymore

## Impacted Areas in Application

`npm t`
